pkg/nuget/csproj/testdata/dev_dependency.csproj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
@Sjord can you help me understand - can we use `TargetFramework` value for condition? [condition](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#adding-a-packagereference-condition)?

Reply:
No. Realistically, a [csproj that has a condition on `TargetFramework`](https://github.com/JamesNK/Newtonsoft.Json/blob/master/Src/Newtonsoft.Json/Newtonsoft.Json.csproj) would also define multiple supported target frameworks. The exact target framework that is currently being built is one of those, but which one is not specified.

Reply:
I see. Thanks! I understand now.